src/rpc/handlers/Ledger.cpp

```diff
@@ -46,6 +46,18 @@ doLedger(Context const& context)
         diff = params.at("diff").as_bool();
     }
 
+    if (params.contains(JS(full)) && params.contains(JS(accounts)))
+    {
+        if (!(params.at(JS(full)).is_bool()) &&
+            !(params.at(JS(accounts)).is_bool()))
+            break;
+
+        else if (
+            (params.at(JS(full)).as_bool() &&
+             params.at(JS(accounts)).as_bool()) == false)
+            return Status{RippledError::rpcNOT_SUPPORTED, "notSupported"};
```

Comment:
You don't need to pass "notSupported" as second argument - it should auto-fill the message from rippled codebase.